Document external PostgreSQL earlier in the installation UX (#3783)
theforeman/foreman-documentation: guides/common/modules/con_postgresql-as-an-external-database-considerations.adoc

* If there is latency between the {Project} server and database server, performance can suffer

ifdef::satellite[]
If you suspect that the PostgreSQL database on your {Project} is causing performance problems, see https://access.redhat.com/solutions/2780871[Satellite 6: How to enable postgres query logging to detect slow running queries] to determine if you have slow queries.

> 

Author: Yes, it was originally a fairly big "block of text", so splitting this up helps visual clarity and "scannability".

Queries that take longer than one second are typically caused by performance issues with large installations, and moving to an external database might not help.
If you have slow queries, contact Red Hat Support.
endif::[]

.Next steps

> If you want your {Project} deployment to use an external PostgreSQL database, do one of the following:
> 
> * Change the database setting when installing {Project}. For details, see {InstallingServerDocURL}configuring-dns-dhcp-and-tftp_{project-context}Configuring_Installation_{project-context}[Configuring {Project} installation]

Reviewer: ```suggestion
If you want your {Project} deployment to use an external PostgreSQL database, do one of the following:

* Change the database setting when installing {Project}.
For more information, see {InstallingServerDocURL}configuring-dns-dhcp-and-tftp_{project-context}Configuring_Installation_{project-context}[Configuring {Project} installation] in _{InstallingServerDocTitle}_.
```

suggestion: split line by sentence; use "For more information"; add name of the guide in italic; end with dot. All to align to our current conventions/style/pattern.

Just FYI: For Every XyzDocURL, there's a XyzDocTitle attribute. General rule: if we link to another guide, we mention the guide. We don't do this when we use  `xref:`.

> * Migrate to the external database on an existing {Project} deployment

Reviewer: ```suggestion
* Migrate your {ProjectServer} to use the external database.
```

suggestion (non-blocking): Switch the order? Your call.

> {AdministeringDocURL}Migrating_from_Internal_Databases_to_External_Databases_admin[Migrating from internal {Project} databases to external databases] in _{AdministeringDocTitle}_.

Reviewer: ```suggestion
For more information, see {AdministeringDocURL}Migrating_from_Internal_Databases_to_External_Databases_admin[Migrating from internal {Project} databases to external databases] in _{AdministeringDocTitle}_.
```
